Ginger/Ginger/ApplicationModelsLib/POMModels/PomElementsPage.xaml.cs

```diff
@@ -922,18 +923,19 @@ private void TestElementButtonClicked(object sender, RoutedEventArgs e)
                 }
                 else if (WorkSpace.Instance.Solution.GetTargetApplicationPlatform(mPOM.TargetApplicationKey).Equals(ePlatformType.Web))
                 {
-                    var htmlElementInfo = new HTMLElementInfo() { 
-                        Path = testElement.Path, 
-                        Locators = testElement.Locators, 
+                    var htmlElementInfo = new HTMLElementInfo() {
+                        Path = testElement.Path,
+                        Locators = testElement.Locators,
                         Properties = ((HTMLElementInfo)CurrentEI).Properties,
                     };
 
                     htmlElementInfo.FriendlyLocators = testElement.FriendlyLocators;
-                    testElement = htmlElementInfo; 
+                    testElement = htmlElementInfo;
                 }
-
-
-                mWinExplorer.TestElementLocators(testElement, false, mPOM);
+                
+                Task.Run(() => {
```

Comment:
Why we need to specifically need to run it on Task?
Also add try catch inside the Task.Run

Reply:
When the user tries to test the locators of an element that does not exist. Selenium takes a long time to respond with a failure, which freezes the Ginger screen and the user is not able to navigate across the application

Reply:
ok, please add the try-catch inside task

Reply:
Implemented the requested